src/uproot/interpretation/objects.py

```diff
@@ -137,7 +126,15 @@ def basket_array(
 
         output = None
         if isinstance(library, uproot.interpretation.library.Awkward):
-            form = self.awkward_form(branch.file, index_format="i64")
+            form = self.awkward_form(
+                branch.file,
+                {
+                    "index_format": "i64",
+                    "header": False,
+                    "tobject_header": True,
+                    "breadcrumbs": (),
+                },
+            )
```

Comment:
This is the one and only place where the `context` for `awkward_form` gets set (that won't be going away, as `uproot.lazy` will).